### PR TITLE
Add orthographic projection class

### PR DESCRIPTION
**Pull request by tommygebhardt**
Add the orthographic projection as a
Math Transform following the algorithms in
proj4, EPSG Guidance note 7.2 and Snyder's map projections

### CODE REVIEW

File: src/ProjNet/ProjNET.csproj [hunk 1]
```diff
@@ -30,7 +30,6 @@ Proj.NET performs point-to-point coordinate conversions between geodetic coordin
   </ItemGroup>
 
 	<ItemGroup Condition=" '$(EnableApiCompat)' == 'true' ">
-		<PackageReference Include="Microsoft.DotNet.ApiCompat" Version="6.0.0-beta.21159.11" PrivateAssets="All" />
```
Comment (FObermaier) [inline src/ProjNet/ProjNET.csproj]: Why did you remove this? Have you tried to update the package?
The functionality it provides should definetly stay in place.

**Review by FObermaier — CHANGES_REQUESTED** [review]

File: src/ProjNet/ProjNET.csproj (hunk 1 above)
Comment (DGuidi) [reply src/ProjNet/ProjNET.csproj]: I can just assume that he experienced the same problem described [here](https://github.com/NetTopologySuite/NetTopologySuite/pull/580#discussion_r781839363). anyway, same reply is still valid.

Comment (tommygebhardt) [reply src/ProjNet/ProjNET.csproj]: Yup, @DGuidi is right. In order to get the project to compile, I had to remove the reference. I didn't mean to include it in this commit though, that's my mistake.
I can't find that nuget package anywhere on nuget.org, including pre-releases. Where do I restore that package from?

Comment (airbreather) [reply src/ProjNet/ProjNET.csproj]: Good catch, it will need to already have been in your package cache in order for this to work.

We should have added a NuGet.config file like what NTS has: https://github.com/NetTopologySuite/NetTopologySuite/blob/54ff4b3a21ded301159d12277cf8515b463f3394/NuGet.config#L1-L6

**Review by FObermaier — APPROVED** [review]

**Comment by FObermaier** [comment]
Thanks! Good start.
